rollup/circuitcapacitychecker/mock.go

```diff
@@ -43,6 +43,10 @@ func (ccc *CircuitCapacityChecker) ApplyBlock(traces *types.BlockTrace) (*types.
 	}}, nil
 }
 
+func (ccc *CircuitCapacityChecker) GetTxNum() (uint64, error) {
+	return 0, nil
```

Comment:
We'll see constant errors when running with mock ccc

Reply:
Fixed to return [uint64(ccc.countdown)](https://github.com/scroll-tech/go-ethereum/pull/505/commits/6387d3f2b22e0928ff7ab62e53742ab687457747#diff-914b7258328693ea76e3080bf0d057bdf98257e3651a24ba9f8b2773aef436f7R47) in commit [6387d3f](https://github.com/scroll-tech/go-ethereum/pull/505/commits/6387d3f2b22e0928ff7ab62e53742ab687457747).

Reply:
We should not use `ccc.countdown`, it is only used in tests. But when our partners build l2geth using `make nccc_geth` and run it, they will also run the mock ccc.

Reply:
I will take care of this PR @silathdiir

Reply:
I fix to add `tcount` to mock CCC and a new function `SetTxNum` to this `mock.go` in commit [a99fc44](https://github.com/scroll-tech/go-ethereum/pull/505/commits/a99fc443402b75abb3462af7ec42c6218c97fa39).

Please correct me if I misunderstood your comment, thanks.